pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -228,6 +246,19 @@ func getProfileVersion(config *checkconfig.CheckConfig) uint64 {
 	return profileVersion
 }
 
+func getDeviceType(store *metadata.Store) string {
+	deviceType := strings.ToLower(store.GetScalarAsString("device.type"))
+	if deviceType == "" {
+		return "other"
+	}
+	_, isValidType := supportedDeviceTypes[deviceType]
+	if isValidType {
+		return deviceType
+	}
```

Comment:
```suggestion
	if supportedDeviceTypes[deviceType] {
		return deviceType
	}
```
Could be simplified like this ☝️ ?